microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -121,6 +123,7 @@ The following are some of the flags we've seen used.
 | `msFloatyShouldHonorIndiaHoldout` | If `true`, honors the India holdout group.  Use this flag, set to `false`, to disable the Floaty feature that's enabled if the user is part of the India holdout group, because WebView doesn't support browser retention experiments. |
 | `msOverlayScrollbarWinStyle` | Whether the users can change between overlay and non-overlay modes for Fluent scrollbars. |
 | `msPdfEnableAsPreview` | This features enables the PDF viewer to launch with a minimal toolbar and in read-only preview mode. |
+| `msSingleSignOnForInPrivateWebView2` | This flag enables the single sign-on (SSO) flow for incognito/private session<!-- todo: InPrivate? --> of WebView2.  Browser-based SSO should seamlessly work in incognito<!-- todo: InPrivate? --> mode after enabling this flag. |
```

Comment:
Fulltext search of Docs repo for 'incognito':
![image](https://github.com/user-attachments/assets/0188dbaa-403f-4b94-b976-88afccd85ffd)
In browser-features.md, check usage there, change to InPrivate there?

Reply:
results:
```
| `incognito` | Forces InPrivate (Incognito) mode even if the user data directory is specified by using the `--user-data-dir` flag. |

| `msSingleSignOnForInPrivateWebView2` | This flag enables the single sign-on (SSO) flow for InPrivate (Incognito) sessions of WebView2.  Browser-based SSO should seamlessly work in InPrivate (Incognito) mode after enabling this flag. |
```
"should" is bad, entering separate PR Comment

Reply:
result:
```
| `msSingleSignOnForInPrivateWebView2` | This flag enables the single sign-on (SSO) flow 
for InPrivate (Incognito) sessions of WebView2.  Enables browser-based SSO in InPrivate 
(Incognito) mode. |
```